otelcollector/shared/configmap/mp/tomlparser-scrape-interval.go

```diff
@@ -93,7 +95,8 @@ func processConfigMap() map[string]string {
 		"WINDOWSKUBEPROXY_SCRAPE_INTERVAL", "PROMETHEUS_COLLECTOR_HEALTH_SCRAPE_INTERVAL",
 		"POD_ANNOTATION_SCRAPE_INTERVAL", "KAPPIEBASIC_SCRAPE_INTERVAL",
 		"NETWORKOBSERVABILITYRETINA_SCRAPE_INTERVAL", "NETWORKOBSERVABILITYHUBBLE_SCRAPE_INTERVAL",
-		"NETWORKOBSERVABILITYCILIUM_SCRAPE_INTERVAL",
+		"NETWORKOBSERVABILITYCILIUM_SCRAPE_INTERVAL", "ACSTORCAPACITYPROVISIONER_SCRAPE_INTERVAL",
```

Comment:
Can you pick these 2 new environment vars, and add them to be part of our telemetry in telemetry.go (under fluent-bit/src) ? This will help us to determine if these targets are enabled for any given cluster where managed prom is enabled. I would also add your keeplists to telemetry, to see what other metrics of  ours, customers are adding to the defaults, so that way we can add to our defaults in the future as needed. This telemetry is also in the same file as in the above comment...

Reply:
Added the env variables and keep lists to `telemetry.go`